mm2src/coins/qrc20/swap.rs

```diff
@@ -346,7 +346,23 @@ impl Qrc20Coin {
             receiver,
             secret_hash,
             ..
-        } = try_s!(self.erc20_payment_details_from_tx(&tx).await);
+        } = try_s!(
+            retry_on_err!(async { self.erc20_payment_details_from_tx(&tx).await })
+                .until_ready()
+                .repeat_every_secs(check_every)
+                .until_s(wait_until)
+                .inspect_err({
+                    let tx_hash = tx.hash().reversed();
+                    move |e| {
+                        error!(
+                            "Failed to retrieve QRC20 payment details from transaction {} \
+                            after waiting until {}. Error: {:?}",
```

Comment:
we didn't necessarily `wait until` the full period here, we should remove this part from the log.

PS.: the second part of this method (`wait_for_tx_spend_impl`) now has less overall time to finish within, so we might wanna keep an eye on that in the time being.

Reply:
Fixed the error message here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2249/commits/e568aed68ca4a30246206dd0105799b7a3939cb6